packages/core/contracts/merkle-distributor/implementation/MerkleDistributor.sol

```diff
@@ -142,17 +130,33 @@ contract MerkleDistributor is Ownable, Testable {
         for (uint256 i = 0; i < claims.length; i++) {
             Claim memory claim = claims[i];
             require(claim.account == account, "Invalid account in batch claim");
-            _markClaimed(claim);
+            _verifyAndMarkClaimed(claim);
             amountToClaim = amountToClaim.add(claim.amount);
         }
-        _disburse(IERC20(rewardToken), account, amountToClaim);
+        IERC20(rewardToken).safeTransfer(account, amountToClaim);
     }
 
     // Claim `amount` of reward tokens for `account`. If `amount` and `account` do not exactly match the values stored
     // in the merkle proof for this `windowIndex` this method will revert.
     function claimWindow(Claim memory claim) public {
-        _markClaimed(claim);
-        _disburse(merkleWindows[claim.windowIndex].rewardToken, claim.account, claim.amount);
+        _verifyAndMarkClaimed(claim);
```

Comment:
nit: should we name this function just `claim`?

Reply:
Yeah less verbose is better

Reply:
Saves 50 gas too 🤨

Reply:
haha